dashboard/src/actions/charts.test.tsx

```diff
@@ -71,7 +72,54 @@ describe("fetchCharts", () => {
       throw new Error("something went wrong");
     });
     axiosWithAuth.get = axiosGetMock;
-    await store.dispatch(actions.charts.fetchCharts(cluster, namespace, "foo"));
+    await store.dispatch(actions.charts.fetchCharts(cluster, namespace, "foo", ""));
+    expect(store.getActions()).toEqual(expectedActions);
+  });
+});
+
+describe("fetchChartCategories", () => {
```

Comment:
Not at all. Indeed this change was introduced and merged in https://github.com/kubeapps/kubeapps/pull/2246. I'm removing it since it has been accidentally duplicated during the merge.